### PR TITLE
Simplify the descriptions for all modules

### PR DESCRIPTION
**Pull request by lasers**
This is a part of an ongoing project, https://github.com/ultrabug/py3status/issues/667, designed to put out a spectacular display of...
# py3status: an extensible i3status wrapper written in python

py3status is an extensible i3status wrapper written in python.

Intelligence design and very mystery.

py3status to impress our audience.

python. overwhelming power and strength.

### CODE REVIEW

File: py3status/modules/battery_level.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the battery level.
+Display battery level.
```
Comment (tobes) [inline py3status/modules/battery_level.py]: Display battery information.

File: py3status/modules/group.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Group a bunch of modules together and switch between them.
+Group and switch between statuses.
```
Comment (tobes) [inline py3status/modules/group.py]: Group modules and switch between them.

File: py3status/modules/mpris.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display information about the current song and video playing on player with
-mpris support.
+Display song/video currently playing in mpris compatible players.
```
Comment (tobes) [inline py3status/modules/mpris.py]: Display song/video playing and control mpris compatible players.

File: py3status/modules/rainbow.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Add color cycling fun to your i3bar.
+Colorize your statuses like a rainbow.
```
Comment (tobes) [inline py3status/modules/rainbow.py]: I prefer the old description.

File: py3status/modules/volume_status.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display current sound volume.
+Display sound volume.
```
Comment (tobes) [inline py3status/modules/volume_status.py]: Display and control sound volume.

File: py3status/modules/window_title.py [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the current window title.
+Display window title.
```
Comment (tobes) [inline py3status/modules/window_title.py]: This needs improving maybe just revert to old one - Also module could do with a description.

File: py3status/modules/xrandr.py [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Control your screen(s) layout easily.
+Control your screen layout easily.
```
Comment (tobes) [inline py3status/modules/xrandr.py]: revert, this module is really about multi screen

Comment (lasers) [inline py3status/modules/xrandr.py]: I looked it up. It seems like people don't use `screens layout`. It's either `Control your screens easily` or `Control your screen layout easily.` I avoid (s) to look clean.

File: py3status/modules/diskdata.py [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display advanced disk usage information
+Display advanced disk space usage.
```
Comment (tobes) [inline py3status/modules/diskdata.py]: `Display disk information.`

it is not just space info

File: py3status/modules/rainbow.py [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Colorize your statuses like a rainbow.
+Colorize your modules.
```
Comment (lasers) [inline py3status/modules/rainbow.py]: This okay? or... Colorize your modules like a rainbow?... Revert?

File: py3status/modules/gpmdp.py [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display currently playing song from Google Play Music Desktop Player.
+Display song currently playing in Google Play Music Desktop Player
```
Comment (tobes) [inline py3status/modules/gpmdp.py]: no full stop?

File: py3status/modules/imap.py [hunk 11]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # FIXME new_mail_color param
 """
-Display the unread messages count from your IMAP account.
+Display total number of unread messages from your IMAP account.
```
Comment (tobes) [inline py3status/modules/imap.py]: is total needed?

File: py3status/modules/kdeconnector.py [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*
 """
-Display information of your android device over KDEConnector.
+Retrieve notifications and others from your smartphone using KDEConnector.
```
Comment (tobes) [inline py3status/modules/kdeconnector.py]: Display information about your smartphone with KDEConnector.

File: py3status/modules/rainbow.py [hunk 13]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Add color cycling fun to your i3bar.
+Colorize your modules.
```
Comment (tobes) [inline py3status/modules/rainbow.py]: Color cycling fun.

File: py3status/modules/scratchpad_async.py [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the amount of windows and indicate urgency hints on scratchpad (async).
+Display total number of windows and indicate urgency hints on your i3 scratchpad asynchronously.
```
Comment (tobes) [inline py3status/modules/scratchpad_async.py]: Display scratchpad status asynchronously.

File: py3status/modules/scratchpad_counter.py [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the amount of windows in your i3 scratchpad.
+Display total number of windows in your i3 scratchpad.
```
Comment (tobes) [inline py3status/modules/scratchpad_counter.py]: why total?

File: py3status/modules/window_title_async.py [hunk 16]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the current window title with async update.
+Display window title asynchronously.
```
Comment (tobes) [inline py3status/modules/window_title_async.py]: Display current window title asynchronously.

File: py3status/modules/xrandr_rotate.py [hunk 17]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 """
-Switch between horizontal and vertical screen rotation on a single click.
+Control screen rotation easily.
```
Comment (tobes) [inline py3status/modules/xrandr_rotate.py]: Control screen rotation.

File: py3status/modules/window_title.py [hunk 18]
```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 """
-Display the current window title.
+Display window title frequently.
```
Comment (tobes) [inline py3status/modules/window_title.py]: Display current window title.

File: py3status/modules/scratchpad_counter.py (hunk 15 above)
Comment (lasers) [inline py3status/modules/scratchpad_counter.py]: Yeah. A tad too much. No `total`.

File: py3status/modules/rainbow.py (hunk 13 above)
Comment (lasers) [inline py3status/modules/rainbow.py]: 😄

File: py3status/modules/scratchpad_async.py (hunk 14 above)
Comment (lasers) [inline py3status/modules/scratchpad_async.py]: Display number of windows and indicate urgency hints on scratchpad asynchronously. (?) ... No urgency hints on `scratchpad_counter`.

File: py3status/modules/gpmdp.py (hunk 10 above)
Comment (lasers) [inline py3status/modules/gpmdp.py]: Are we talking about a closing period? Bad deletion. If not, what's it?

Comment (tobes) [inline py3status/modules/gpmdp.py]: `.`  <-- this thing

Comment (lasers) [inline py3status/modules/gpmdp.py]: Yeah. Bad deletion. Fixed. See `iii`. Ty for replying back.

File: py3status/modules/scratchpad_async.py (hunk 14 above)
Comment (tobes) [inline py3status/modules/scratchpad_async.py]: this is still overly long.

File: py3status/modules/diskdata.py [hunk 19]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display advanced disk usage information
+Display advanced disk usage.
```
Comment (tobes) [inline py3status/modules/diskdata.py]: why advanced?

Comment (lasers) [inline py3status/modules/diskdata.py]: Not me.

Comment (tobes) [inline py3status/modules/diskdata.py]: kill the advanced too

**Comment by lasers** [comment]
Touch or leave...
A simple "Do Not Disturb" module that can turn on and off all system notifications. ??

File: py3status/modules/dpms.py [hunk 20]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Activate or deactivate DPMS and screen blanking.
+Control your DPMS and screen blanking easily.
```
Comment (tobes) [inline py3status/modules/dpms.py]: `Control your DPMS screen blanking.`

DPMS is screen blanking

File: py3status/modules/fedora_updates.py [hunk 21]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-The number of package updates pending for a Fedora Linux installation.
+Display number of pending updates for Fedora Linux.
```
Comment (tobes) [inline py3status/modules/fedora_updates.py]: Display pending updates for Fedora Linux.

same with arch_updates

File: py3status/modules/dpms.py [hunk 22]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Activate or deactivate DPMS and screen blanking.
+Control DPMS and screen blanking.
```
Comment (tobes) [inline py3status/modules/dpms.py]: Control DPMS screen blanking.

DMPS = screen blanking

File: py3status/modules/scratchpad_async.py [hunk 23]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the amount of windows and indicate urgency hints on scratchpad (async).
+Display number of windows in scratchpad asynchronously and indicate urgency hints.
```
Comment (tobes) [inline py3status/modules/scratchpad_async.py]: I still think this is overly long

File: py3status/modules/twitch_streaming.py [hunk 24]
```diff
@@ -1,5 +1,5 @@
 """
-Checks if a Twitch streamer is online.
+Follow Twitch streamer with this status notification.
```
Comment (tobes) [inline py3status/modules/twitch_streaming.py]: Checks if a Twitch streamer is online.

was much better imho

File: py3status/modules/scratchpad_async.py (hunk 23 above)
Comment (lasers) [inline py3status/modules/scratchpad_async.py]: Display number of windows in scratchpad and indicate urgency hints. (async) 
And put (async) on window_title_async.py too to balance it out?

Comment (tobes) [inline py3status/modules/scratchpad_async.py]: Display scratchpad status asynchronously.

File: py3status/modules/dpms.py (hunk 22 above)
Comment (lasers) [inline py3status/modules/dpms.py]: I think they're different things, but we're turning both of them on/off.

File: py3status/modules/scratchpad_async.py (hunk 23 above)
Comment (lasers) [inline py3status/modules/scratchpad_async.py]: Can't. Urgency hint. Important feature. `vi` pushed. I think we're done!

File: py3status/modules/fedora_updates.py (hunk 21 above)
Comment (lasers) [inline py3status/modules/fedora_updates.py]: I like to keep `number of pending updates`. I already changed many other things. I think we did great! Time for break? Bbl. 👍

Comment (lasers) [inline py3status/modules/fedora_updates.py]: Come up with something for...

A simple "Do Not Disturb" module that can turn on and off all system notifications. ??
Take a screenshot and optionally upload it to your online server. ??

Brb soon! Making coffee.

File: py3status/modules/spaceapi.py [hunk 25]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Follow your favorite hackerspace with this status notification.
+Display status on your favorite hackerspace.
```
Comment (tobes) [inline py3status/modules/spaceapi.py]: on --> of

**Comment by tobes** [comment]
Display is used many many times is this something needed?

Anyhow let me know when you have finished with your changes and I'll do a final review.

**Comment by lasers** [comment]
It's pretty much finished. Just couple of slow tweaking and final touches. `py3status modules list` looks trimmed. No more long lines. 👍

**Comment by tobes** [comment]
> Just couple of slow tweaking and final touches. 

Fine, let me know when you have completely finished.

**Comment by lasers** [comment]
It's finished. Won't touch it anymore.

File: py3status/modules/do_not_disturb.py [hunk 26]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-A simple "Do Not Disturb" module that can turn on and off all system notifications.
+Do not disturb by turning on and off all system notifications.
```
Comment (tobes) [inline py3status/modules/do_not_disturb.py]: Turn on and off dbus notifications.

File: py3status/modules/file_status.py [hunk 27]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display if a file or dir exists.
+Display status for file or directory.
```
Comment (tobes) [inline py3status/modules/file_status.py]: I think the original `Display if a file or dir exists.` was a better description  `status` implies last modified, permissions etc imho

File: py3status/modules/process_status.py [hunk 28]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display if a process is running.
+Display status of a given process.
```
Comment (tobes) [inline py3status/modules/process_status.py]: Again I think the old description is better `Display if a process is running.`  status implies running/zombie niceness etc

File: py3status/modules/do_not_disturb.py (hunk 26 above)
Comment (lasers) [inline py3status/modules/do_not_disturb.py]: Should we keep "Do Not Disturb" to help people recognize it for what it is? From Apple. https://support.apple.com/en-us/HT204321

File: py3status/modules/file_status.py (hunk 27 above)
Comment (lasers) [inline py3status/modules/file_status.py]: What if file or dir does not exist? Does it stop display? Idk.

File: py3status/modules/do_not_disturb.py (hunk 26 above)
Comment (tobes) [inline py3status/modules/do_not_disturb.py]: Fuck apple ;p

File: py3status/modules/file_status.py (hunk 27 above)
Comment (tobes) [inline py3status/modules/file_status.py]: No it just shows if it exists or not

Comment (lasers) [inline py3status/modules/file_status.py]: Will change this and process_status. I found couple more. Sry. Pushing out now.

File: py3status/modules/do_not_disturb.py (hunk 26 above)
Comment (lasers) [inline py3status/modules/do_not_disturb.py]: Saw your question.
>Display is used many many times is this something needed?

I saw that too in the `list`. Crazy. Idk. We could fix that later.

**Comment by tobes** [comment]
Cool let's leave it here.

Thanks for this major bit of cleanup

**Comment by lasers** [comment]
Np. Done amending `do_not_disturb`. Ty for looking at them.

File: py3status/modules/dropboxd_status.py [hunk 29]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display dropboxd status.
+Display service status of Dropbox daemon.
```
Comment (lasers) [inline py3status/modules/dropboxd_status.py]: Does this make sense? `Display service status for Dropbox` maybe? 
EDIT: No... We want Dropbox daemon, not the service itself. Will leave it alone, mate.

**Comment by ultrabug** [comment]
Thanks a lot @lasers, I'm eager to see the work on modules doc with screenshots coming live too ;)

**Pull request merged**